src/main.cpp

```diff
@@ -111,7 +112,7 @@ void setup() {
     watchFaceSwitcher.registerApp(new OswAppWatchfaceBinary());
     watchFaceSwitcher.registerApp(new OswAppWatchfaceMonotimer());
     mainAppSwitcher.registerApp(&watchFaceSwitcher);
-
+    mainAppSwitcher.registerApp(new OswAppWeather());
```

Comment:
Nope, this call should be moved down to the other `mainAppSwitcher.registerApp()` calls. This is because we want the watch to start as fast as possible and initializing all the apps before the first frame it somewhat bad... So we are initializing them after the first frame is drawn and visible instead.